sqlglot/dialects/starrocks.py

```diff
@@ -14,11 +14,18 @@
 )
 from sqlglot.dialects.mysql import MySQL
 from sqlglot.helper import seq_get
+from sqlglot.tokens import TokenType
 
 
 class StarRocks(MySQL):
     STRICT_JSON_PATH_SYNTAX = False
 
+    class Tokenizer(MySQL.Tokenizer):
+        KEYWORDS = {
+            **MySQL.Tokenizer.KEYWORDS,
+            "PARTITION BY RANGE": TokenType.PARTITION_BY_RANGE,
+        }
+
```

Comment:
Let's not do it like this, remove the token and try to capture this property using `_match_text_seq`. Lately we've been avoiding polluting the token enum, mostly adding new tokens for types etc.

This also has the problem of assuming only 1 space between the keywords, which isn't necessarily true, e.g. `PARTITION     BY` is also valid right? This problem exists in some other parts of the codebase today, but we're probably going to refactor those as well at some point.

Reply:
Where do I change this part? Currently this is for keywords. Do you have any code reference?

Reply:
I'd look in the base parser's `PROPERTY_PARSERS` dict and probably extend/override the parser corresponding to `PARTITION BY`.

Reply:
Fixed